packages/shared-ux/page/analytics_no_data/src/page.component.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { KibanaNoDataPage } from '@kbn/shared-ux-components';
```

Comment:
My understanding was we would move `KibanaNoDataPage` to its own package first. I guess eventually it's going to be just changing an import?

Reply:
That's right.  Rather than postpone *this* PR, we can change the import internally in parallel to teams adopting the page.